features/mbedtls/importer/Makefile

```diff
@@ -157,7 +157,7 @@ update: $(MBED_TLS_GIT_CFG)  $(MBED_TLS_HA_GIT_CFG)
 	git -C $(MBED_TLS_DIR) checkout $(MBED_TLS_RELEASE)
 	#
 	# Updating checked out version tag
-	echo $(MBED_TLS_RELEASE) > $(TARGET_PREFIX)VERSION.txt
+	git -C $(MBED_TLS_DIR) describe --tags --abbrev=12 --dirty --always > $(TARGET_PREFIX)/VERSION.txt
```

Comment:
Is the addition of the `/` between TARGET_PREFIX and VERSION intentional?

Reply:
Nope. Fixed.